napari_animation/_qt/animation_widget.py

```diff
@@ -145,6 +150,21 @@ def _save_callback(self, event=None):
         print('Saving animation to', path)
         self.animation.animate(path)
 
+    def _update_theme(self, event=None):
+        """Update from the napari GUI theme"""
+        from napari.utils.theme import get_theme, template
+
+        # get theme and raw stylesheet from napari viewer
+        theme = get_theme(self.viewer.theme)
+        raw_stylesheet = self.viewer.window.qt_viewer.raw_stylesheet
+
+        # template and apply the primary stylesheet
+        templated_stylesheet = template(raw_stylesheet, **theme)
+        self.setStyleSheet(templated_stylesheet)
+
+        # update styling of KeyFramesListWidget
+        self.keyframesListWidget._update_theme(theme)
+
     def close(self):
         self._release_callbacks()
```

Comment:
Can you add here inside the close method `self.viewer.events.theme.disconnect(self._update_theme)` too